src/pages/RunEncounter/DisplayInitiative.tsx

```diff
@@ -56,6 +65,30 @@ const DisplayInitiative: React.FunctionComponent = () => {
   const initiativeCreatures = useAppSelector(selectSortedInitiativeCreatures);
   const dispatch = useAppDispatch();
 
+  const onDragEnd = (result: DropResult) => {
+    // dropped outside the list
+    if (!result.destination) {
+      return;
+    }
+
+    dispatch(reorderCreature({ index: result.source.index, newIndex: result.destination.index }));
+  };
+
+  const getItemStyle = (
+    isDragging: boolean,
+    draggableStyle: NotDraggingStyle | DraggingStyle | undefined
+  ): React.CSSProperties => ({
+    // some basic styles to make the items look a bit nicer
+    userSelect: "none",
+    marginBottom: "1rem",
+
+    // styles we need to apply on draggables
```

Comment:
I feel like the name is descriptive enough that you don't need this comment

Reply:
Addressed